src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -525,10 +526,13 @@ public Response datasetAutocomplete(@Auth AuthUser authUser,
   @Path("/search")
   @Produces("application/json")
   @PermitAll
-  public Response searchDatasets(@Auth AuthUser authUser, @QueryParam("query") String query) {
+  public Response searchDatasets(
+      @Auth AuthUser authUser,
+      @QueryParam("query") String query,
+      @QueryParam("open") Boolean openAccess) {
```

Comment:
This needs a default value
```suggestion
      @QueryParam("open") @DefaultValue("false") Boolean openAccess) {
```